.teamcity/settings.kts

```diff
@@ -801,165 +801,3 @@ object WpCalypso : GitVcsRoot({
 	}
 })
 
-object WpDesktop : Project({
-	name = "Desktop app"
-	buildType(WpDesktop_DesktopE2ETests)
-
-	params {
-		text("docker_image_dekstop", "registry.a8c.com/calypso/ci-desktop:latest", label = "Docker image", description = "Docker image to use for the run", allowEmpty = true)
```

Comment:
Note this typo (`dekstop`) has been fixed in `.teamcity/_self/projects/DesktopApp.kt`